src/desktop/apps/auction/components/DOM.js

```diff
@@ -46,9 +48,29 @@ class DOM extends Component {
     this.$registerBtn.off("click", this.handleRegister)
   }
 
+  showModal = type => {
+    this.props.dispatch(showModal(type))
+  }
+
+  maybeShowModal() {
+    const { pathname } = location
+    console.log(pathname)
+    if (pathname.match(/registration-flow$/) && this.props.me) {
+      this.showModal("RegistrationFlow")
+    } else if (pathname.match(/\/confirm-registration/) && this.props.me) {
+      this.showModal(
+        location.search.match("origin=bid")
+          ? "ConfirmBidAndRegistration"
+          : "ConfirmRegistration"
+      )
+    }
+  }
+
   handleRegister = event => {
+    console.log("handleRegister")
     const { auction, me } = this.props
     // If there is no user, log in and redirect to this flow
+    // debugger
```

Comment:
stray

Reply:
I'll clean these up before taking the WIP label off, but am only posting this for early feedback on things like general approach/antipatterns etc right now.